eslint.config.mjs

```diff
@@ -53,6 +53,7 @@ export default tseslint.config(
         "error",
         { assertionStyle: "never" },
       ],
+      "@typescript-eslint/no-misused-spread": "off",
```

Comment:
@tdresser I wanted to flag a change I made to the ESLint config. This rule blocks dangerous spreads, including class objects. While it would be useful to block things like functions and strings, I couldn't find a way to allow only class objects to be spread.

https://github.com/gpo/grassroots/blob/5799501c5506c9ca4e0087e061ac5f4a3cbf60af/grassroots-backend/src/contacts/contacts.service.spec.ts#L52-L54

Reply:
You can always use instanceToPlain from class transformer, and then spread that.

Reply:
ok, sounds good. Let's put this back in